README.md

```diff
@@ -4,21 +4,21 @@
 
 --------------
 
-
-* [Overview](#overview)
-* [Downloads](#downloads)
-* [Changelog](#changelog)
-* [Compilation](#compilation)
-* [FFmpeg vs. Libav](#ffmpeg-vs-libav)
-* [Release cycle](#release-cycle)
-* [Bug reports](#bug-reports)
-* [Contributing](#contributing)
-* [Relation to MPlayer and mplayer2](#relation-to-mplayer-and-mplayer2)
+* [#Overview](#overview)
+* [#System requirements](#system-requirements)
+* [#Downloads](#downloads)
+* [#Changelogs](#changelogs)
+* [#Compilation](#compilation)
+* [#FFmpeg vs. Libav](#ffmpeg-vs-libav)
+* [#Release cycle](#release-cycle)
+* [#Bug reports](#bug-reports)
+* [#Contributing](#contributing)
+* [#Relation to MPlayer and mplayer2](#relation-to-mplayer-and-mplayer2)
+* [#Contact](#contact)
+* [#License](#license)
 * [Wiki](https://github.com/mpv-player/mpv/wiki)
 * [FAQ](https://github.com/mpv-player/mpv/wiki/FAQ)
-* [Man pages](http://mpv.io/manual/master/)
-* [Contact](#contact)
-* [License](#license)
+* [Man pages](https://mpv.io/manual/master/)
```

Comment:
Should be `manual`, since it's not actually just 'man pages' or even one 'man page'.